rest-api-spec/src/main/resources/rest-api-spec/test/cat.nodes/10_basic.yml

```diff
@@ -58,6 +58,32 @@
       $body: |
                /^      http \n ((\d{1,3}\.){3}\d{1,3}:\d{1,5}\n)+  $/
 
+  - do:
```

Comment:
Will move the tests to the new section "Additional disk information". Should I add the skip attribute with the given values (version, reason) too?